draft-ietf-quic-transport.md

```diff
@@ -2525,9 +2526,6 @@ a small packet might result in Stateless Reset not being useful in detecting
 cases of broken connections where only very small packets are sent; such
 failures might only be detected by other means, such as timers.
 
-An endpoint can increase the odds that a packet will trigger a Stateless Reset
```

Comment:
Why did you remove this sentence?

Reply:
See #2153.  Or were you asking for it to be retained?

Reply:
Yes. I think it's a good idea to always send a packet padded to > 40 bytes after quiescence, so you'd be able to quickly recover by receiving the stateless reset.

Reply:
It might be nice to explicitly mention the packet after quiescence as part of this advice, and/or advise sending at least one 40+ byte packet in each flight.